assets/src/edit-story/components/popup/index.js

```diff
@@ -65,31 +65,7 @@ const Container = styled.div.attrs(
   position: fixed;
   z-index: 2;
 
-  /*
-   * Custom gray scrollbars for Chromium & Firefox.
-   */
-  * {
-    scrollbar-width: thin;
-    scrollbar-color: ${({ theme }) =>
-      `${theme.DEPRECATED_THEME.colors.bg.v10} ${theme.DEPRECATED_THEME.colors.bg.v12}`};
-  }
-
-  *::-webkit-scrollbar {
-    width: ${SCROLLBAR_WIDTH}px;
-    height: ${SCROLLBAR_WIDTH}px;
-  }
-
-  *::-webkit-scrollbar-track {
-    background: ${({ theme }) => theme.DEPRECATED_THEME.colors.bg.v12};
-  }
-
-  *::-webkit-scrollbar-thumb {
-    background-color: ${({ theme }) => theme.DEPRECATED_THEME.colors.fg.v5};
-    border: 2px solid ${({ theme }) => theme.DEPRECATED_THEME.colors.bg.v12};
-    border-left-width: 3px;
-    border-top-width: 3px;
-    border-radius: 6px;
-  }
+  ${scrollBarStyles}
```

Comment:
This is just cleaning up the code but not changing the colors of the scrollbar yet? Should it adjust the scrollbar colors, too? Or do we have a separate issue for that?

Edit: just saw this: https://github.com/google/web-stories-wp/issues/6285

Reply:
Exactly. It's actually also implementing the dark scrollbars, as the scrollbar in here was in light theme. Not really sure why, maybe we have an inverted-color popup somewhere earlier with scroll, but I don't think we do anymore.